.pre-commit-config.yaml

```diff
@@ -3,13 +3,19 @@ repos:
     rev: v4.5.0
     hooks:
       - id: check-merge-conflict
+        exclude: ^examples/
```

Comment:
I think maybe we shouldn't exclude these? I know we talked about this yesterday, don't remember what we settled on